File: Jack.Model/Model/Roupa.vb
<Serializable()>
Public Class Roupa

    Public Sub New()

        Codigo = 0
        Tamanho = String.Empty
        Idade = 0
        MedidaIdade = String.Empty

    End Sub

    Public Overridable Property Codigo As Integer
    Public Overridable Property Tamanho As String
    Public Overridable Property Idade As Integer
    Public Overridable Property MedidaIdade As String

    Public Overridable ReadOnly Property IdadeString As String
        Get
            If MedidaIdade = "M" Then
                Return Idade.ToString() + " meses"
            Else
                Return Idade.ToString() + " anos"
            End If
        End Get
    End Property

End Class

File: Jack.Model/Model/Roupa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Jack.Model
{
    [Serializable()]
    public class Roupa: BaseModel<Roupa>
    {


        public Roupa()
        {
            Codigo = 0;
            tamanho = string.Empty;
            idade = 0;
            medidaIdade = string.Empty;

        }

        private string tamanho;
        private int idade;
        private string medidaIdade;

        public virtual string Tamanho
        {
            get
            {
                return tamanho;
            }
            set
            {
                tamanho = value;
            }
        }

        public virtual int Idade
        {
            get
            {
                return idade;
            }
            set
            {
                idade = value;
            }
        }

        public virtual string MedidaIdade
        {
            get
            {
                return medidaIdade;
            }
            set
            {
                medidaIdade = value;
            }
        }

        public virtual string IdadeString
        {
            get
            {
                if (MedidaIdade == "M")
                {
                    return Idade.ToString() + " meses";
                }
                else {
                    return Idade.ToString() + " anos";
                }
            }
        }

    }

}
